### ISSUE TITLE
The label of the Native Inputs is not accessible and screen readers don't read it

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-vue#93]
## I'm submitting a...
* Bug report 

## Current behavior
The labels of the Native inputs are not pronounced by screen readers like NVDA. 

## Expected behavior
The labels of the Native inputs should be pronounced by the screen readers. 

## Minimal reproduction of the problem with instructions

1. Open [https://www.telerik.com/kendo-vue-ui/components/inputs/input/forms/](https://www.telerik.com/kendo-vue-ui/components/inputs/input/forms/)
2. Start the NVDA screen reader
3. Select an Input field in the first example. The label of the selected field will not be announced.

**Comment by zdravkov** [comment]
Fixed in 2.1.2
